src/main/java/bio/terra/common/ValidationUtils.java

```diff
@@ -109,6 +133,24 @@ public static LinkedHashMap<String, String> validateRelationshipTerm(
     return termErrors;
   }
 
+  private static Optional<ColumnModel> retrieveColumnModelFromTerm(
+      RelationshipTermModel term, List<TableModel> tables) {
+    String tableName = term.getTable();
+    String columnName = term.getColumn();
+    try {
+      return tables.stream()
+          .filter(t -> t.getName().equals(tableName))
+          .findFirst()
+          .get()
+          .getColumns()
+          .stream()
+          .filter(c -> c.getName().equals(columnName))
+          .findFirst();
+    } catch (NoSuchElementException ex) {
+      return Optional.empty();
+    }
```

Comment:
You can avoid the `try` / `catch` here by using `flatMap()` on the first optional. In general, I would avoid using `.get()` on an optional directly if possible.
```suggestion
    return tables.stream()
        .filter(t -> t.getName().equals(tableName))
        .findFirst()
        .flatMap(tableModel ->
            tableModel.getColumns()
                .stream()
                .filter(c -> c.getName().equals(columnName)).findFirst()
        );
```